include/boost/ut.hpp

```diff
@@ -274,17 +284,23 @@ inline constexpr const std::string_view need_name =
     "unique_name_for_auto_detect_prefix_and_suffix_lenght_0123456789_struct";
 #endif
 inline constexpr const std::size_t need_length = need_name.length();
+#if not defined(BOOST_UT_USE_MODULE)
```

Comment:
Note this is not great. I think to solve this properly the selection of which declarations to export and not to export needs a rework. Either exporting in a smaller scope than the entire `boost::ut` namespace or using `using namespace::declaration` at the end of the header.

Reply:
I would +1 the explicit symbol export, as it also causes troubles when building with MSVC, see [here](https://github.com/boost-ext/ut/pull/604/files#diff-38970d6ec46c6be8f51a9f82b79acafebab04dc54e8024c17fee20a2313393b6R2763) for an example.

Reply:
I'm happy with that too :point_up: